### PR TITLE
Declare TESTS_TO_RUN as a global variable

### PR DESCRIPTION
**Pull request by medb**
Fixes "TESTS_TO_RUN[@]: unbound variable" error

### CODE REVIEW

File: cloudbuild/presubmit.sh [hunk 1]
```diff
@@ -67,7 +67,7 @@ determine_tests_to_run() {
   echo "Changed directories: ${changed_dirs[*]}"
 
   # Determines what tests in changed init action directories to run
-  declare -a TESTS_TO_RUN
+  declare -g -a TESTS_TO_RUN
```
Comment (functicons) [inline cloudbuild/presubmit.sh]: Why put it inside of the function?

Comment (medb) [reply cloudbuild/presubmit.sh]: This is how function passes result.

Do you think will be better to declare it as a global variable at the top of the script?

Comment (functicons) [reply cloudbuild/presubmit.sh]: Yes.

Comment (medb) [reply cloudbuild/presubmit.sh]: Done

**Pull request merged**